jwst/datamodels/container.py

```diff
@@ -271,7 +273,7 @@ def read_asn(filepath):
 
     def from_asn(self, asn_data):
         """
-        Load fits files from a JWST association file.
+        Load FITS files from a JWST association file.
```

Comment:
Looking at this function I don't think it needs to be FITS, it could be ASDF too.

Also, I don't really see a use-case for this method outside of `__init__`, should we consider underscore-protecting it?

Reply:
I can't comment on `read_asn` but I think it has similar problems